### PR TITLE
add remove to EMQDeque

### PR DESCRIPTION
**Pull request by sideshowdave7**
What:
cleanup_dead_workers relies on this being available

Tests:
nosetests pass

### CODE REVIEW

File: eventmq/utils/classes.py [hunk 1]
```diff
@@ -426,6 +426,9 @@ def append(self, item):
             self._queue.append(item)
             return True
 
+    def remove(self, item):
+        return self._queue(item)
```
Comment (com4) [inline eventmq/utils/classes.py]: Why doesn't it remove the item?  self._queue.remove(item)

**Comment by com4** [comment]
Now that the tests are working, I think we should start testing at the very least new code we write. We'll have to go back and add tests where they are missed later.

**Comment by sideshowdave7** [comment]
Ooops.  Yea I will write a test for this and re submit

**Comment by sideshowdave7** [comment]
Added a bunch of testing, should be good to go now

**Pull request merged**